docs/howto/operate/k8s/index.md

```diff
@@ -0,0 +1,25 @@
+# Work with Kubernetes infrastructure
+
+There are some cases where you need to inspect or modify actively-running Kubernetes infrastructure, or make changes to the Kubernetes cluster on which our hubs are running.
+This is a more manual step than [modifying a hub's configuration](/topic/config) and requires some extra tools, setup, and expertise.
+These sections cover how to do this.
+
+## Get started
+
+First off, you'll need to gain access to the Kubernetes cluster.
+See the following section for instructions on how to do so.
+
+```{toctree}
+:maxdepth: 2
+
+cmd-access.md
```

Comment:
Currently, this section talks specifically about GCP-based clusters, in the future we may add the AWS-specific stuff, the Azure stuff, etc...

Reply:
good point - I'll add a note at the top about this